tests/lora/utils.py

```diff
@@ -129,6 +129,34 @@ class PeftLoraLoaderMixinTests:
     text_encoder_target_modules = ["q_proj", "k_proj", "v_proj", "out_proj"]
     denoiser_target_modules = ["to_q", "to_k", "to_v", "to_out.0"]
 
+    cached_non_lora_outputs = {}
+
+    @pytest.fixture(scope="class", autouse=True)
```

Comment:
We use setUp and tearDown convention instead of fixtures. I would stick with that for consistency.

Reply:
To handle multiple test classes in a single file we can use a dict with keys based on the testing pipeline class name.

Reply:
> To handle multiple test classes in a single file we can use a dict with keys based on the testing pipeline class name.

Didn't get it.

> We use setUp and tearDown convention instead of fixtures. I would stick with that for consistency.

The issue is in how pytest handles test discovery and execution, especially when mixing it with unittest-style setups. When you run a single test method directly, pytest doesn't automatically execute the setUpClass method. We often run individual tests (or even a group of tests) for better debugging. So, not sure how setup and teardown would work in this setup.